src/ray/common/ray_config.h

```diff
@@ -63,7 +63,16 @@ class RayConfig {
   inline type &name() { return name##_; }
 
 #include "ray/common/ray_config_def.h"
-/// -------------------------------------------------------------------------
+
+/// -----------Include ray_process_flag_def.h to define internal flags-------
```

Comment:
How about RAY_INTERNAL_FLAG? From the comment, it's also saying it's internal flags I think?

My feeling is that RAY_PROCESS_FLAGS doesn't give a clear explanation about that it's for internal use and don't use it externally.

Reply:
Sorry, I missed your previous comments. I think something INTERNAL there looks better.

Reply:
Sounds good. Will update to INTERNAL